test/test_cli.py

```diff
@@ -16,38 +42,54 @@ def test_token_issue_verbose():
     runner = CliRunner()
-    result = runner.invoke(cli.cli, ['--verbose', 'token', 'get', '--uid', 'FA1C76805DD54FDBBC8945F5B5F514C7'])
+    result = runner.invoke(cli.cli, ['token', 'revoke', '--refreshtoken', 'https://cilogon.org/oauth2/refreshToken/4884e9b835260111384cf83c2617efbf/1604534366554'])
     print(result.output)
-    assert result.exit_code == 1
-    print(result.exception)
+    assert result.exit_code != 0
 
 
-def test_token_get_success():
+def test_resources_get():
     runner = CliRunner()
-    result = runner.invoke(cli.cli, ['token', 'issue'])
+    result = runner.invoke(cli.cli, ['resources', 'query'])
     print(result.output)
-    getcommand = []
-    for line in result.output.split('\n'):
-        if 'fabric-cli token get --uid' in line:
-            getcommand = line.split()[1:]
-            break
-    input("\nLogin at the URL, then Press Enter to continue...\n")
-    result = runner.invoke(cli.cli, getcommand)
+    assert result.exit_code != 0
+
+
+def test_resources_get_with_id_token():
+    runner = CliRunner()
+    result = runner.invoke(cli.cli, ['resources', 'query', '--idtoken',
+                                     'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJlbWFpbCI6Imt0aGFyZTEwQGVtYWlsLnVuYy5lZHUiLCJnaXZlbl9uYW1lIjoiS29tYWwiLCJmYW1pbHlfbmFtZSI6IlRoYXJlamEiLCJuYW1lIjoiS29tYWwgVGhhcmVqYSIsImlzcyI6Imh0dHBzOi8vY2lsb2dvbi5vcmciLCJzdWIiOiJodHRwOi8vY2lsb2dvbi5vcmcvc2VydmVyQS91c2Vycy8xMTkwNDEwMSIsImF1ZCI6ImNpbG9nb246L2NsaWVudF9pZC8xMjUzZGVmYzYwYTMyM2ZjYWEzYjQ0OTMyNjQ3NjA5OSIsInRva2VuX2lkIjoiaHR0cHM6Ly9jaWxvZ29uLm9yZy9vYXV0aDIvaWRUb2tlbi8zNzY5NDgzNTcxNGM5MjUyNDc3M2MxYmYxOGVkOWVjYS8xNjA0NTQyMTI0OTU2IiwiYXV0aF90aW1lIjoiMTYwNDU0MjEyNCIsImV4cCI6MTYwNDU0NTc0NiwiaWF0IjoxNjA0NTQyMTQ2LCJyb2xlcyI6WyJDTzptZW1iZXJzOmFjdGl2ZSIsIkNPOkNPVTpKdXB5dGVyaHViOm1lbWJlcnM6YWN0aXZlIiwiQ086Q09VOnByb2plY3QtbGVhZHM6bWVtYmVyczphY3RpdmUiXSwic2NvcGUiOiJhbGwiLCJwcm9qZWN0IjoiYWxsIn0.V_tQCx97g5KS4dJH20f7XtIHyRQIu5U0yhmxLs8R9DMXhWZn51f8bBGogisYivs3bGXHIsR3YwJP8GEFJbjyZRFKyj9RgnqWhttug3UmbL09AgqtZs2mbLfmFIZGub-5wreM625GLX4el8IdxrUfV1H8W5o0b-7cCA7n006lRZI0L0rbZiKVJNBhuXn8vV8KdgLaKKQ-ixL1zTfi_-g7bvPXy6FEOxBgNeDETb9svmidjGneuexuH66xa8MTrYSj0gfXDHRC__MzAGNkfCpCKMcUc0kNlPtXCpoa-fVgY2Lw3qEbD16BhHMHvSNQdAyFIcL7UvRXxs7OPEsz8KLObA'])
     print(result.output)
-    assert result.exit_code == 0
-    assert 'id_token' in result.output
-    assert 'refresh_token' in result.output
+    assert result.exit_code != 0
+
+
+def test_resources_get_with_refresh_token():
+    runner = CliRunner()
+    result = runner.invoke(cli.cli, ['resources', 'query', '--refreshtoken',
+                                     'https://cilogon.org/oauth2/refreshToken/5ff57e742945eed9b2377b3f123ecc8d/1604543301530'])
+    print(result.output)
+    assert result.exit_code != 0
+
 
+if __name__ == '__main__':
+    import os
+    os.environ['FABRIC_CREDMGR_HOST'] = 'dev-2.fabric-testbed.net'
+    os.environ['FABRIC_ORCHESTRATOR_HOST'] = 'dev-2.fabric-testbed.net'
+    #test_token_issue()
```

Comment:
Is it commented by purpose to just test the last one case

Reply:
Was trying to test the last scenario, so left it there, will uncomment the rest of tests